connector/routingconnector/internal/common/functions.go

```diff
@@ -19,6 +19,8 @@ func createRouteFunction[K any](_ ottl.FunctionContext, _ ottl.Arguments) (ottl.
 func Functions[K any]() map[string]ottl.Factory[K] {
 	return ottl.CreateFactoryMap(
 		ottlfuncs.NewIsMatchFactory[K](),
+		ottlfuncs.NewIsMapFactory[K](),
+		ottlfuncs.NewIsStringFactory[K](),
 		ottlfuncs.NewDeleteKeyFactory[K](),
 		ottlfuncs.NewDeleteMatchingKeysFactory[K](),
 		// noop function, it is required since the parsing of conditions is not implemented yet,
```

Comment:
```suggestion
func Functions[K any]() map[string]ottl.Factory[K] {
	return ottlfuncs.StandardConverters[K]()
}
```

I could also see this being an implementation choice, I'll leave it up to the maintainers here if they think that is too much!

Reply:
I think the routing connect should have access to all converters. This is safe since converters never change the underlying telemetry itself. @jpkrohling, @mwear @djaglowski do you agree?